submodules/wizard/wizard.js

```diff
@@ -133,72 +133,74 @@ define(function(require) {
+						callback: waterfallCallback
+					});
+				},
+				function tryGetResellerAccount(parentAccount, waterfallCallback) {
+					var results = {
+							parentAccount: parentAccount,
+							servicePlans: []
 						},
-						function tryGetResellerAccount(parentAccount, waterfallCallback) {
-							var results = {
-									parentAccount: parentAccount,
-									servicePlans: []
-								},
-								resellerAccountId = self.wizardGetResellerAccountId(parentAccount);
+						resellerAccountId = self.wizardGetResellerAccountId(parentAccount);
 
-							self.wizardRequestGetAccount({
-								accountId: resellerAccountId,
-								generateError: false,
-								callback: function(err) {
-									var status = _.get(err, 'status'),
-										isResellerUnavailable = _.includes([ 403, 404 ], status);
+					if (resellerAccountId === parentAccountId) {
+						self.wizardSetStore('resellerAccountId', parentAccountId);
 
-									if (!isResellerUnavailable) {
-										self.wizardSetStore('resellerAccountId', resellerAccountId);
-									}
+						return waterfallCallback(null, _.merge({
+							isResellerUnavailable: false
+						}, results));
+					}
 
-									return waterfallCallback(null, _.merge({
-										isResellerUnavailable: isResellerUnavailable
-									}, results));
-								}
-							});
-						},
-						function getServicePlans(results, waterfallCallback) {
-							var isResellerUnavailable = results.isResellerUnavailable,
-								isCurrentAccountReseller = monster.util.isReseller(),
-								isCurrentUserSuperDuperAdmin = monster.util.isSuperDuper(),
-								skipServicePlans = isResellerUnavailable || !(isCurrentAccountReseller || isCurrentUserSuperDuperAdmin);
-
-							if (skipServicePlans) {
-								return waterfallCallback(null, results);
+					self.wizardRequestGetAccount({
+						accountId: resellerAccountId,
+						generateError: false,
+						callback: function(err) {
+							var status = _.get(err, 'status'),
+								isResellerUnavailable = _.includes([ 403, 404 ], status);
+
+							if (!isResellerUnavailable) {
+								self.wizardSetStore('resellerAccountId', resellerAccountId);
 							}
 
-							self.wizardGetServicePlanList({
-								success: function(plans) {
-									waterfallCallback(null, _.merge(results, {
-										servicePlans: plans
-									}));
-								},
-								error: function() {
-									waterfallCallback(null, results);
-								}
-							});
+							return waterfallCallback(null, _.merge({
+								isResellerUnavailable: isResellerUnavailable
+							}, results));
 						}
-					];
+					});
+				},
+				function getServicePlans(results, waterfallCallback) {
+					var isResellerUnavailable = results.isResellerUnavailable,
+						isCurrentAccountReseller = monster.util.isReseller(),
+						isCurrentUserSuperDuperAdmin = monster.util.isSuperDuper(),
+						skipServicePlans = isResellerUnavailable || !(isCurrentAccountReseller || isCurrentUserSuperDuperAdmin);
```

Comment:
Maybe we would extract the result of the in-parenthesis check into its own variable to make it easier to read this? Something like `isElevatedAccount`?